features/golden-recognize.js

```diff
@@ -0,0 +1,61 @@
+const config = require("../config");
+const recognition = require("../service/recognition");
+const winston = require("../winston");
+const { SlackError, GratitudeError } = require("../service/errors");
+const { userInfo } = require("../service/apiwrappers");
+const {
+  handleSlackError,
+  handleGratitudeError,
+  handleGenericError,
+  sendNotificationToReceivers,
+} = require("../service/messageutils");
+
+const { goldenRecognizeEmoji } = config;
+
+module.exports = function (app) {
+  app.message(goldenRecognizeEmoji, respondToRecognitionMessage);
+};
+
+async function respondToRecognitionMessage({ message, client }) {
+  winston.info(`Heard reference to ${goldenRecognizeEmoji}`, {
+    callingUser: message.user,
+    slackMessage: message.text,
+  });
+  let gratitude;
+  try {
+    gratitude = {
+      giver: await userInfo(client, message.user),
+      receivers: await Promise.all(
+        recognition
+          .gratitudeReceiverIdsIn(message.text)
+          .map(async (receiver) => userInfo(client, receiver))
+      ),
+      count: 25,
+      message: message.text,
+      trimmedMessage: recognition.trimmedGratitudeMessage(message.text),
+      channel: message.channel,
+      tags: recognition.gratitudeTagsIn(message.text),
+      type: goldenRecognizeEmoji,
+    };
+
+    await recognition.validateAndSendGratitude(gratitude);
```

Comment:
At a first glace, I'm concerned that granting the golden fistbumps points using the existing recognition collection might have unintended ramifications.

One example might be that giving away the golden fistbump might instantly deplete your normal stock of fistbumps for the day.

Has that been accounted for?

Reply:
An alternative might be to use the new golden fistbump collection to store the new values. But it would likely require some changes to at least the functionality of 'balance'

Reply:
I accounted for being able to give 25 fistbumps but I believe it will currently deplete the normal fistbumps as well and put a users daily balance in the negatives. Good catch. I think the balance function will need to be updated. 

The way I was thinking of implementing this is to add a `ignoreForBalance` attribute to the recognition collections which signals to ignore that specific recognition when calculating that users balance. Fistbumps given as part of a golden fistbump will mark this attribute as true, defaults to false

Reply:
might be easy enough to just combine golden fistbumps and normal fistbumps for a total though 🤔

Reply:
@Pactionly I implemented the goldenrecognition to be worth a seperate value from a normal recognition which addresses the issues talked about above. Ready for another look